repositories/patches/ros2cli_replace-netifaces.patch

```diff
@@ -0,0 +1,74 @@
+diff --git ros2cli/package.xml ros2cli/package.xml
+index a7a858b..b3e1a4a 100644
+--- ros2cli/package.xml
++++ ros2cli/package.xml
+@@ -18,9 +18,9 @@
+ 
+   <exec_depend>python3-argcomplete</exec_depend>
+   <exec_depend>python3-importlib-metadata</exec_depend>
+-  <exec_depend>python3-netifaces</exec_depend>
+   <exec_depend>python3-packaging</exec_depend>
+   <exec_depend>python3-pkg-resources</exec_depend>
++  <exec_depend>python3-psutil</exec_depend>
```

Comment:
Oh, well, we don't need this, but I guess you wanted to recycle the patch from https://github.com/ros2/ros2cli/pull/875.